### PR TITLE
#M14. Misc cleanup changes and improvements

### PR DESCRIPTION
**Pull request by stefdworschak**
## Description
Fixing error with scoring and changing team members during testing.

## Pull request type
- [x] #M01 (Miscellaneous User Story #01)
- [ ] #P02 (Participant User Story #02)
- [ ] #S03 (Staff User Story #03)
- [ ] #A04 (Admin User Story #04)
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Related Issue
#131

### CODE REVIEW

File: left_to_do.txt [hunk 1]
```diff
@@ -1 +1,3 @@
 - Add notification to notify participants of winners
+- Check if teams can submit projects still if status is judging, finished, deleted
```
Comment (TravelTimN) [inline left_to_do.txt:2]: Teams should only be able to submit projects until a deadline.... not if the status is judging, finished, or deleted.... that would royally mess up the scoring, no? Ideally, like deployment, get the project submitted initially, and can edit where needed up until the deadline to submit... no late submissions, once it's changed over to "Judging", done, no more submissions permitted (ie: you snooze, you lose).

**Review by TravelTimN — APPROVED** [review]
LGTM - one minor comment only

File: left_to_do.txt (hunk 1 above)
Comment (stefdworschak) [reply left_to_do.txt:2]: Okay, sure. Just was not sure we wanted to be that strict :smile:

Comment (stefdworschak) [reply left_to_do.txt:2]: I did add this in to only allow teams to submit in status `hack_in_progress`.

**Review by TravelTimN — COMMENTED** [review]
LGTM!

**Pull request merged**